api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -38,4 +40,8 @@
       @Param("dals") List<Short> dataAccessLevels, @Param("term") String term, Sort sort);
 
   Set<DbUser> findByFirstRegistrationCompletionTimeNotNull();
+
+  default Map<Boolean, Long> getDisabledToCountMap() {
+    return DaoUtils.getAttributeToCountMap(findAll(), DbUser::getDisabled);
```

Comment:
We can do that. That does sound more interesting.

Thinking out loud, if I want to put multiple dimensions attachments on an object, that's definitely doable, but now I'm in the business of scanning through all of them and making a different bundle for each tuple in the cartesian product of the labels. Since we're talking about single digit thousands of users, I think we can do this fast enough.

Reply:
I have a prototype of a multidimensional categorization of users.